packages/main/src/models/conversation.ts

```diff
@@ -109,6 +109,9 @@ export function getPresentableElements({
         isRead: cache
           .getFlags(latestEdit ? latestEdit.revision.message.id : message.id)
           .includes("\\Seen"),
+        isStarred: cache
+          .getFlags(latestEdit ? latestEdit.revision.message.id : message.id)
+          .includes("\\Flagged"),
```

Comment:
This will be handy when we display stars per presentable. But I would like this to be computed differently: if you star a message the star should remain when the message is edited. And it it should be possible to a remove a star at the presentable level even if the presentable has been edited, which is only possible if the star is displayed. In other words a presentable should be considered starred if any of its revisions is starred.

This is the sort of behavior that is especially useful to write tests for - what happens to state when these things happen? It's kinda like board game rules: you don't want situations to come up where the game rules are genuinely ambiguous on what should happen. A game designer should try to anticipate that kind of thing, and make sure that the game rules are comprehensive enough to cover all situations. In an application tests should be comprehensive enough to specify what should happen in situations where a feature might reasonably be implemented in different ways.

But like I said, this computation is not currently used because there is no per-presentable star control yet. I added a note in "This Week" to add that feature.